api/turing/server/application.go

```diff
@@ -117,6 +119,19 @@ func Run() {
 	// HealthCheck Handler
 	AddHealthCheckHandler(r, "/v1/internal", db)
 
+	// Override the bundle file with user specified overrides.
+	if spaCfg := cfg.OpenapiConfig.SwaggerUIConfig; spaCfg != nil && len(spaCfg.ServingDirectory) > 0 {
```

Comment:
Why are user-specified overrides only applied when `SwaggerUIConfig` is configured? Openapi spec file is not directly related to the swagger UI. Swagger UI is just a convenience added on top of the spec file

Reply:
Removed and reworked.